sagemaker-spark-sdk/build.sbt

```diff
@@ -30,10 +30,10 @@ version := {
 
 libraryDependencies ++= Seq(
   "org.apache.hadoop" % "hadoop-aws" % "2.8.1",
```

Comment:
you'll probably need to update `hadoop-aws` as well

Reply:
I saw that in your PR, but I found it wasn't necessary to upgrade the AWS Java SDK and unblock the PRs when testing locally.

Reply:
FYI, build failure was expected because it was pulling the old version of the IT repo without the fix I just committed. I wanted to see a failure before a success to verify my fix.

Reply:
Hah... This next one will also fail because I forgot to merge the IT change...
Third time's the charm!

Reply:
Additionally, it looks like the ITs were pointed towards Jonathan's branch. I updated that and things are looking good.